### ISSUE TITLE
Update Versions 

### ISSUE THREAD

**Issue by kcarros** [GoogleCloudPlatform/cloudml-edge-automation#2]
In the **requirements.txt** file:

```
apache-beam[gcp]==2.2.0
pillow
tensorflow==1.3.0

```

Updates: 

**apache-beam[gcp]==2.4.0**
pillow
tensorflow==1.3.0
**protobuf==3.5.1**
**google-cloud-pubsub==0.26.0
google-api-core==1.1.2
google-cloud-bigquery==0.25.0
google-cloud-core==0.25.0**

[pip_dependencies.txt](https://github.com/GoogleCloudPlatform/cloudml-edge-automation/files/1980724/pip_dependencies.txt)

**Comment by ptone** [comment]
Thanks for this report, are you saying you were able to work through the tutorial after pinning to the above versions?

**Comment by kcarros** [comment]
Yes, updating the versions as suggested did allow the code to run successfully.
